meta/sai_meta.cpp

```diff
@@ -6333,10 +6333,10 @@ void meta_sai_on_fdb_flush_event_consolidated(
     // and find fdb entries
 
     auto bpid = sai_metadata_get_attr_by_id(SAI_FDB_ENTRY_ATTR_BRIDGE_PORT_ID, data.attr_count, data.attr);
-    auto type = sai_metadata_get_attr_by_id(SAI_FDB_ENTRY_ATTR_TYPE, data.attr_count, data.attr);
+    auto flush_fdb_type_attr_p = sai_metadata_get_attr_by_id(SAI_FDB_ENTRY_ATTR_TYPE, data.attr_count, data.attr);
```

Comment:
Was there any function changes here, same thing for below few changes?  If not, I would rather remove them from this PR itself, and send out another improvement PR, probably upstream it.  We don't keep any private code like this and make the merge challenge later.